apps/contract/scripts/deploy-auth.sh

```
@@ -0,0 +1,135 @@
+    PUBLIC_KEY_HEX="${PUBLIC_KEY_HEX:0:130}"
+fi
+
+echo "Using device_id hash: $DEVICE_ID_HASH"
+echo "Using public_key (hex): $PUBLIC_KEY_HEX"
+echo "Public key length: ${#PUBLIC_KEY_HEX} characters (should be 130)"
+
+# Deploy Account Contract with constructor arguments
+# ? credential_id (device_id) from devices off-chain table: must be the admin who can update the contract config
+# ? public_key from devices off-chain table: must be the admin who can update the contract config
+ACCOUNT_CONTRACT_ID=$(stellar contract deploy \
+    --network $NETWORK \
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)